### PR TITLE
Integrate the SDK changes, use SORV2 from the SDK

### PR DESCRIPTION
**Pull request by danielmkm**
# Description

Integrate with the proposed `balancer-sdk` changes. The SOR is now exported from the sdk, so it can be removed from the `package.json`. 

Please don't merge this PR, it is just to showcase what needs to be done for integration. The SDK needs to be redeployed to NPM prior to merging.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other

## How should this be tested?

The app should function as normal, this is simply a refactor to use the SOR and `Network` exported from the `balancer-sdk`.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] My changes generate no new console warnings
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/composables/useNetwork.ts [hunk 1]
```diff
@@ -1,17 +1,5 @@
...
-export enum Network {
-  MAINNET = 1,
-  ROPSTEN = 3,
-  RINKEBY = 4,
-  GÖRLI = 5,
-  KOVAN = 42,
-  POLYGON = 137,
-  ARBITRUM = 42161
-}
+import { Network } from '@balancer-labs/sdk';
```
Comment (garethfuller) [inline src/composables/useNetwork.ts:2]: Seems a bit weird to pull the network type from the SDK. I know the types will most likely be the same, but I think the network types available in the app should probably be defined in the app. I don't feel strongly about this, it's just a thought, feel free to keep it if you think it's the best path.

**Review by garethfuller — COMMENTED** [review]
This is just a preliminary review, once the SDK/SOR PRs are finalised I will test and review again.

In general, looks good so far. I can't help but feel the network type change is a little unnecessary and actually adds a lot of changes to this PR that aren't needed, e.g. all the network type imports.

**Comment by garethfuller** [comment]
@danielmkm is this ready to be updated with a new SDK release?

**Comment by danielmkm** [comment]
I've merged develop into this branch and updated the SDK to use the latest npm version. Tested that everything still compiles correctly and app runs fine.

**Review by garethfuller — APPROVED** [review]

**Pull request merged**